src/launch_common.py

```diff
@@ -43,19 +43,14 @@ def print_banner(args, script_name):
     with open("./banner.txt", "rt") as file:
         print(file.read())
     print(LINER, flush=True)
-    print("Copyright Huseyin ABANOZ 2019")
+    print("Copyright Huseyin ABANOZ 2021")
```

Comment:
We need to either adopt this to  the whole TRD group...

Reply:
@jdsika why do you think this is necessary?